spyder/plugins/outlineexplorer/api.py

```diff
@@ -243,3 +255,17 @@ def has_name(self):
             return True
         else:
             return False
+
+    def blockNumber(self):
```

Comment:
Is this an redefined Qt method? If so, please mention it in the docstring below, explaining why we need to do that. If not, please rename it to `get_block_number`.